tchannel/response.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from . import transport
+
+__all__ = ['Response']
+
+
+class Response(object):
+    def __init__(self, header, body, transport):
+        self.header = header
```

Comment:
consistent。 If we use header, then header everywhere. If headers, headers everywhere.